aero.minova.core.application.system.setup/src/main/java/ch/minova/install/setup/BaseSetupException.java

```diff
@@ -0,0 +1,8 @@
+package ch.minova.install.setup;
+
+@SuppressWarnings("serial")
+public class BaseSetupException extends Exception {
+	public BaseSetupException(final String format) {
+		super(format);
```

Comment:
Falls irgendeine der Exceptions geloggt werden soll, müssten sie in dem ControllerExceptionHandler noch hinzugefügt werden.

Reply:
Gibt es hier was generisches?
Sollte es nicht möglich sein ein Default-Exception-Handler zu erstellen?

Reply:
Hatten wir bisher nicht, weil wir unterscheiden mussten, ob der Client eine Table oder ein ProcedureResultSet erwartet.
Aber da das hier nur den CAS etwas angeht, könnte man noch einen Default-Exception-Handler erstellen.

Reply:
Ich merke gerade, dass die Exceptions der Extensions jeweils in einer ProcedureException gepackt werden.
Ansonsten, ist mir es jetzt zu aufwändig einen Default-Exception-Handler zu erstellen.